capellambse_context_diagrams/collectors/tree_view.py

```diff
@@ -337,8 +337,10 @@ def _make_class_info(
     multiplicity = None
     target = None
     if prop is not None:
-        start = converter.get(prop.min_card.value, str(prop.min_card.value))
-        end = converter.get(prop.max_card.value, str(prop.max_card.value))
+        min_card = getattr(prop.min_card, "value", 1.0)
+        max_card = getattr(prop.max_card, "value", 1.0)
+        start = converter.get(min_card, str(min_card))
+        end = converter.get(max_card, str(max_card))
```

Comment:
Using a floating point number for the min/max_card looks a bit odd, especially since the other hunk uses (stringified) integer numbers. Are you sure this is correct?

Also note that (for a while now) the `value` field of `LiteralNumericValue` is actually returned as a string anyways (which will be `"*"` for infinity), so we can probably get rid of this whole "converter" business as well.